include/filewriter/summarywriter_impl.hpp

```diff
@@ -122,6 +122,131 @@ void SummaryWriter<Filewriter>::Image(const std::string& tag,
     remove(fileNames[i].c_str());
 }
 
+template<typename Filewriter>
+void SummaryWriter<Filewriter>::PrCurve(const std::string tag,
+                                        const std::vector<double>& labels,
+                                        const std::vector<double>& predictions,
+                                        Filewriter& fw,
+                                        int threshold,
+                                        std::vector<double>weights,
+                                        const std::string& displayName,
+                                        const std::string& description)
+{
+    // Pr plugin
+    mlboard::PrCurvePluginData *prCurvePlugin = new PrCurvePluginData();
+    prCurvePlugin->set_version(0);
+    prCurvePlugin->set_num_thresholds(threshold);
+    std::string pr_curve_content;
+    prCurvePlugin->SerializeToString(&pr_curve_content);
+
+    // PluginMeta data
+    mlboard::SummaryMetadata_PluginData *pluginData =
+        new SummaryMetadata::PluginData();
+    pluginData->set_plugin_name("pr_curves");
+    pluginData->set_content(pr_curve_content);
+
+    // Summary Meta data
+    mlboard::SummaryMetadata *metaData = new SummaryMetadata();
+    metaData->set_display_name(displayName == "" ? tag : displayName);
+    metaData->set_summary_description(description);
+    metaData->set_allocated_plugin_data(pluginData);
+
+    // misbheaves when thresholds is greater than 127
```

Comment:
```suggestion
    // Misbehaves when thresholds is greater than 127.
```

Just read the comment, is this a common problem? Maybe it makes sense to elaborate on that comment, to make it less confusing :)